### ISSUE TITLE
Code: 502 Message: Command APPE not implemented.

### ISSUE THREAD

**Issue by noname9xndz** [robinrodricks/FluentFTP#1154]
FTP Server OS: Unix 

**FTP Server Type:** Vsftpd / glFTPd / BFTPd / ProFTPD / Pure-FTPd / IBM CS FTP (z/OS, OS/400) / Windows CE / Windows Server IIS / Vax / VMS / OpenVMS / Tandem / HP NonStop

Client Computer OS: Ubuntu

FluentFTP Version: 44.0.1

Framework: .NET 6

Hi team, I am having trouble upgrading versions from 32.3.3 to 43.0.1 or 44.0.1
The UploadFile function with streams or bytes has one exception: "Code: 502 Message: Command APPE not implemented." at FluentFTP.FtpClient.UploadStream.
Before this function works, can we support me?

**Comment by robinrodricks** [comment]
Show the logs, show what arguments you are calling the function with.

I think you are calling it with Append config, so its trying to append. Maybe you need another setting.

**Comment by FanDjango** [comment]
@robinrodricks and I have discussed this and here is some information for you:

1. Most of the time, we cannot do very much without a verbose log and some initial work by the user to isolate the problem a little bit. **So, show the logs**.

2. You are coming from a very old version, so the statement 

> Before this function works (sic)

is very fishy. Altough it **does sometimes happen**, things normally get better with the passing of time. **So, something must have changed**. What?

3. Some time ago, someone discovered that the retry functionality of UploadFile was not working correctly, **it was not retrying**. So it was fixed. If you search the issues, you would find it.

4. The retry happens because of a failed upload. It can be seen in the log. Show the log or look at it yourself. The retry calculates the position of failure and uses **APPE** to continue the upload.

5. If your server does not support APPE, it will fail. Your old version did not attempt retry because this was broken, so you never saw this error. We should not attempt APPE if your server does not support this, so maybe we can improve the error exception. We could see in the log if this is or is not the case. **So: Show the logs**. What was the answer to the FEAT command on connect?

6. So much guesswork and the crystal ball is getting cloudy. We wish people would put the logs in right away, it should save so much time and unneccessary guesswork.
